universal-login-relayer/src/config/baseConfig.ts

```diff
@@ -33,4 +33,14 @@ export const baseConfig = {
   emailPassword: getEnv('EMAIL_PASSWORD', ''),
   copyToClipboardUrl: 'https://universal-provider-backend.netlify.app/copy',
   emailLogo: UNILOGIN_LOGO_WITH_NAME_URL,
+  mailingCredentials: {
```

Comment:
we can change mailingCredentials to `email` and put into this variable all email props:
```ts
{
  emailPassword: ..
  emailLogo:
   copyToClipboardUrl
...
```